openmetadata-ui/src/main/resources/ui/src/styles/app.less

```diff
@@ -120,9 +120,10 @@ a[href].link-text-grey,
   border-top-width: 0px;
 }
 .border-t-1 {
-  border-top-width: 1px;
-  border-style: solid;
-  border-color: @border-color;
+  border-top: @global-border;
+}
+.border-r-1 {
```

Comment:
can you move this border utilities to border.less, instead of keeping it in app.less

Reply:
updated!